app.js

```diff
@@ -252,6 +252,7 @@ exports.init = function (isA11yTest = false, a11yTestSession = {}, ftValue) {
     if (['development', 'testing'].includes(config.nodeEnvironment)) {
         const sslDirectory = path.join(__dirname, 'app', 'resources', 'localhost-ssl');
         const sslOptions = {
+            secureProtocol: 'TLSv1_2_method',
```

Comment:
this is deprecated and not recommended do not add,

> secureProtocol <string> Legacy mechanism to select the TLS protocol version to use, it does not support independent control of the minimum and maximum version, and does not support limiting the protocol to TLSv1.3. Use minVersion and maxVersion instead. The possible values are listed as SSL_METHODS, use the function names as strings. For example, use 'TLSv1_1_method' to force TLS version 1.1, or 'TLS_method' to allow any TLS protocol version up to TLSv1.3. It is not recommended to use TLS versions less than 1.2, but it may be required for interoperability. Default: none, see minVersion.

Reply:
Hi Tim, ok I've updated this to use minVersion and maxVersion both against the TLSv1.2 protocol.
minVersion: "TLSv1.2",
maxVersion: "TLSv1.2"

Reply:
Updated to just use minVersion as per Tim's advice.